batch.js

```diff
@@ -85,21 +85,30 @@ const getContract = (id) => {
   // console.log('----- contract abi ',IdentityABI.abi)
   return new web3.eth.Contract(
     IdentityABI.abi,
-    get(ContractsAddress, `fuse.Identity`, IdentityABI.networks['121'].address),
+    get(ContractsAddress, `Identity`, IdentityABI.networks[String(id)].address),
     {
       from: address,
-      gas: 500000,
+      gas: 1000000,
       gasPrice: web3Utils.toWei('1', 'gwei'),
     },
   )
 }
 
 const test = async () => {
-  setProvider('http://localhost:9545')
-  // getFirstWallet('drip industry pizza deny pistol stem must device citizen crowd offer now') // fuse
-  // contract = getContract(121) //fuse
-  getFirstWallet('myth like bonus scare over problem client lizard pioneer submit female collect') // local
-  contract = getContract(4447) //local
+  // FUSE
+  // setProvider('wss://explorer-node.fuse.io/ws')
+  setProvider('https://rpc.fuse.io/')
+  getFirstWallet('drip industry pizza deny pistol stem must device citizen crowd offer now') // fuse
```

Comment:
exposing secrets!

Reply:
trying version web3@1.0.0-beta.55
results:
![image](https://user-images.githubusercontent.com/36693645/65485157-b6d89f80-dea9-11e9-9a9b-fde0e1ac2ccc.png)

Reply:
remove mnemnic.

test your commit with local blockchain - works

Reply:
but it is not working in fuse 
if websocket provider - it is just stuck 
if http provider - have this error:

![image](https://user-images.githubusercontent.com/36693645/65487665-0621ce80-deb0-11e9-9f9e-cef97c86adb5.png)

Reply:
it works with fuse if we add tx to batch like this
```
batch.add(
    contract.methods
      .whiteListUser('0x919b99dcabae3fea4909af23b8dfa9f2d2c273de', 'ab')
      .send({...params,nonce}, (err, res) => console.log('TX 3: ', { err, res }))
  )```
not like this
```
batch.add(
    contract.methods
      .whiteListUser('0x919b99dcabae3fea4909af23b8dfa9f2d2c273de', 'ab')
      .send.request({...params,nonce}, (err, res) => console.log('TX 3: ', { err, res }))
  )
```